.github/workflows/tests-schedule.yml

```diff
@@ -23,6 +23,9 @@ jobs:
       - name: Upgrade system packages
         run: python -m pip install --upgrade pip setuptools wheel
 
+      - name: SSL
+        run: python -c 'import ssl; print(ssl.OPENSSL_VERSION)'
+
```

Comment:
This is still helpful or just some debug info ?

Reply:
I would say useful debug info for the future. Right now we don't know whether it was the runner which updated the SSL version or if something changed for the server.